docs/extensions.md

```diff
@@ -113,7 +113,7 @@ If order or association are not needed, it's also possible to directly convert t
 **Creating and manipulating arrays in Go:**
 
 ```go
-// export_php:function process_data_ordered(array $input): array
+//export_php:function process_data_ordered(array $input): array
```

Comment:
I'm not sure we should change this. The blank space actually doesn't matter with the extension generator, and Goland adds the blank space when saving the file anyway

Reply:
Understood. Changes reverted.